source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -292,9 +293,9 @@ package:
        and/or deserialize a property.
 
    * - ``BsonProperty``
-     - Allows for an alternative document key name when converting the POJO
-       field to BSON. Also allows a field to turn on using a discriminator
-       when storing a POJO value.
+     - Specifies a custom document field name when converting the POJO
+       field to BSON. You can include a discriminator if you need one for
+       serializing nested POJOs.
```

Comment:
Issue: Consider moving the "if" statement to the [beginning of the sentence](https://docs.mongodb.com/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/#use-a-consistent-sentence-structure). 

Comment: This sentence was tough for me to understand. I think changing this from "if you need one for serializing..." to "If you must serialize a nested POJO" could help clarify the use case.

```suggestion
       field to BSON. If you must serialize a field containing a nested POJO, you can include a discriminator to
       specify how this field is to be serialized.
```

Reply:
I'm not sure I see consistent sentence structure in this suggestion -- none of the settings here are using "If... then" as descriptions and don't think that would be an appropriate way to describe them.

Instead, I'll make it consistent with the structure of the previous sentence. "Optionally specifies the value of a discriminator..."

Reply:
Another thought -- most settings/method descriptions contain the following info:
- description of what its effects are
- optional: describes default values and behaviors of other values
- optional: examples
- optional: direction on where to go for more information

I don't think consistent sentence structure across all these types of info is useful, but we could try to stick to the same form for each individual type of information.

In these sentences, it's describing what its effects are (both required and optional values), so keeping the same structure seems like a good idea.

Reply:
Updated sentence to remove "if you need one" which was perhaps the point of confusion.